bin/lutris

```diff
@@ -27,16 +27,35 @@ if os.path.isdir(os.path.join(LAUNCH_PATH, "../lutris")):
 else:
     sys.path.insert(0, os.path.normpath(os.path.join(LAUNCH_PATH, "../lib/lutris")))
 
-try:
-    gettext.textdomain('lutris')
-except:
-    sys.stderr.write("Couldn't set gettext domain, translations won't work.")
-
 try:
     locale.setlocale(locale.LC_ALL, "")
 except locale.Error:
     sys.stderr.write("Unsupported locale setting. Fix your locales\n")
 
+try:
+    # optional_settings does not exist if you don't use the meson build system
+    from lutris import optional_settings
+
+    try:
+        locale.bindtextdomain("lutris", optional_settings.LOCALE_DIR)
+        gettext.bindtextdomain("lutris", optional_settings.LOCALE_DIR)
+    except:
+        sys.stderr.write(
+            "Couldn't bind gettext domain, translations may not work.\n"
+            "LOCALE_DIR: %s\n" % optional_settings.LOCALE_DIR
+        )
+except:
+    sys.stderr.write(
+        "Unable to load optional_settings, locale dir is not set, "
+        "translations may not work.\n"
+    )
+
+try:
+    locale.textdomain("lutris")
+    gettext.textdomain("lutris")
+except:
+    sys.stderr.write("Couldn't set gettext domain, translations won't work.\n")
```

Comment:
I guess this can be merged into one try and except:
```python3
try:
    # optional_settings does not exist if you don't use the meson build system
    from lutris import optional_settings

    try:
        locale.bindtextdomain("lutris", optional_settings.LOCALE_DIR)
        gettext.bindtextdomain("lutris", optional_settings.LOCALE_DIR)
        locale.textdomain("lutris")
        gettext.textdomain("lutris")
    except:
        sys.stderr.write(
            "Couldn't bind gettext domain, translations won't work.\n"
            "LOCALE_DIR: %s\n" % optional_settings.LOCALE_DIR
        )
except ModuleNotFoundError:
    sys.stdout.write(
        "Unable to load locale dir, translations won't work.\n"
    )
```
There is no real need to print out to stderr, since for now Meson isn't the default.

Reply:
> I guess this can be merged into one try and except:
> 
> ```python
> try:
>     # optional_settings does not exist if you don't use the meson build system
>     from lutris import optional_settings
> 
>     try:
>         locale.bindtextdomain("lutris", optional_settings.LOCALE_DIR)
>         gettext.bindtextdomain("lutris", optional_settings.LOCALE_DIR)
>         locale.textdomain("lutris")
>         gettext.textdomain("lutris")
>     except:
>         sys.stderr.write(
>             "Couldn't bind gettext domain, translations won't work.\n"
>             "LOCALE_DIR: %s\n" % optional_settings.LOCALE_DIR
>         )
> except ModuleNotFoundError:
>     sys.stdout.write(
>         "Unable to load locale dir, translations won't work.\n"
>     )
> ```
> 
> There is no real need to print out to stderr, since for now Meson isn't the default.

Done the change. But it should be an `ImportError`:
```
Traceback (most recent call last):
  File "/home/hu60/.local/bin/lutris", line 37, in <module>
    from lutris import optional_settings
ImportError: cannot import name 'optional_settings' from 'lutris' (/home/hu60/.local/lib/python3.7/site-packages/lutris/__init__.py)
```